static/app/views/starfish/views/spanSummaryPage/index.tsx

```diff
@@ -64,24 +63,16 @@ function SpanSummaryPage({params, location}: Props) {
   const {data: spanMetrics, isLoading: isSpanMetricsLoading} = useSpanMetrics(
     groupId,
     queryFilter,
-    [
-      SpanMetricsField.SPAN_OP,
-      SpanMetricsField.SPAN_GROUP,
-      SpanMetricsField.PROJECT_ID,
-      `${SpanFunction.SPS}()`,
-    ],
+    ['span.op', 'span.group', 'project.id', 'sps()'],
```

Comment:
Using plain strings is more readable and dense in a good way. Since the argument is typed, it's not possible to typo, so the enum is not really necessary.

Reply:
https://blog.sentry.io/performance-impact-of-generated-javascript/#switching-from-typescript-enums-to-const-and-string-enums 👀